pymc/model/fgraph.py

```diff
@@ -278,12 +278,18 @@ def fgraph_from_model(
     return fgraph, memo
 
 
-def model_from_fgraph(fgraph: FunctionGraph) -> Model:
+def model_from_fgraph(fgraph: FunctionGraph, mutate_fgraph: bool = False) -> Model:
```

Comment:
Every time you call the function you use `mutate_fgraph = True`, should that be the default instead? Or are the instances in this PR special cases (and there are many others)

Reply:
I prefer to ask for consent